Assets/Scripts/SitelinkMarker.cs

```diff
@@ -81,7 +81,14 @@ public class SitelinkMarker : MonoBehaviour, IPointerEnterHandler, IPointerExitH
         lineColor.a = blinkLevel;
 
         // Redraw to update blink level.
-        Draw();
+        if (fromSiteMarker.gameObject.layer != 12 && toSiteMarker.gameObject.layer != 12)
```

Comment:
This 12 is kind of a magic number, where does it come from? Could we define it as a constant somewhere?